deploy/sample-run-farm-recipes/aws_ec2.yaml

```diff
@@ -0,0 +1,24 @@
+run_farm_type: AWSEC2F1
+args:
+    # REQUIRED: tag to apply to run farm hosts
```

Comment:
If everything is required, maybe we can just remove the "REQUIRED: prefix" put a header comment saying all of the fields below must be defined in a recipe.yaml, but can be selectively overriden in a runtime-config.yaml?

Reply:
+1